client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -3,9 +3,25 @@ import { useSite } from 'calypso/landing/stepper/hooks/use-site';
 import Checklist from './checklist';
 import { tasks } from './tasks';
 
+function getUrlInfo( url: string ) {
+	const urlWithoutProtocol = url.replace( /^https?:\/\//, '' );
+
+	// Ex. mytest.wordpress.com matches mytest
+	const siteName = urlWithoutProtocol.match( /^[^.]*/ );
+	// Ex. mytest.wordpress.com matches .wordpress.com
+	const topLevelDomain = urlWithoutProtocol.match( /\..*/ ) || [];
+
+	return [ siteName ? siteName[ 0 ] : '', topLevelDomain ? topLevelDomain[ 0 ] : '' ];
+}
+
 const Sidebar = () => {
 	const site = useSite();
-	const url = site?.URL?.replace( /^https?:\/\//, '' );
+	let siteName = '';
```

Comment:
We can eventually update this to use the siteSlug prop after we merge https://github.com/Automattic/wp-calypso/pull/66715